src/app/types/tv-show.ts

```diff
@@ -7,10 +7,12 @@ export interface TvShow {
   poster: string | null;
   title: string;
   mediaType: MediaType.Tv;
-  releaseDate?: Date;
+  releaseDate: string;
```

Comment:
Why it's string now? Maybe we can map this property to Date object

Reply:
As it turned out, API returns empty string for some of the movies/ts-shows, transforming which to date with `new Date()` resulted in `Invalid date`. Decided to switch field type to string to avoid dealing with wrong conversion result `Invalid date`.

Reply:
In my opinion, if there is empty string from API, then we could just put undefined for this property

Reply:
You're right, thanks, returned `Date | undefined` as type.